--- mumble-1.2.4/src/mumble/AudioInput.cpp~	2014-01-09 10:15:07.909966051 +0100
+++ mumble-1.2.4/src/mumble/AudioInput.cpp	2014-01-09 10:15:53.780675423 +0100
@@ -696,8 +696,9 @@
 		cCodec->celt_encoder_ctl(ceEncoder, CELT_RESET_STATE);
 
 	cCodec->celt_encoder_ctl(ceEncoder, CELT_SET_PREDICTION(0));
-
+#ifdef CELT_SET_VBR_RATE
 	cCodec->celt_encoder_ctl(ceEncoder, CELT_SET_VBR_RATE(iAudioQuality));
+#endif
 	len = cCodec->encode(ceEncoder, psSource, buffer, qMin(iAudioQuality / (8 * 100), 127));
 	iBitrate = len * 100 * 8;
 
