--- scripts/murmur.ini.orig	2013-06-01 23:16:31.000000000 +0200
+++ scripts/murmur.ini	2014-01-09 10:08:11.826864748 +0100
@@ -13,7 +13,7 @@
 
 # Path to database. If blank, will search for
 # murmur.sqlite in default locations or create it if not found.
-database=
+database=/var/lib/murmur/murmur.sqlite
 
 # If you wish to use something other than SQLite, you'll need to set the name
 # of the database above, and also uncomment the below.
@@ -73,12 +73,14 @@
 # on Unix-like systems, Murmur will force itself into foreground
 # mode which logs to the console.
 #logfile=murmur.log
+logfile=/var/log/murmur
 
 # If set, Murmur will write its process ID to this file
 # when running in daemon mode (when the -fg flag is not
 # specified on the command line). Only available on
 # Unix-like systems.
 #pidfile=
+pidfile=/var/run/murmurd.pid
 
 # The below will be used as defaults for new configured servers.
 # If you're just running one server (the default), it's easier to
@@ -163,6 +165,7 @@
 # If Murmur is started as root, which user should it switch to?
 # This option is ignored if Murmur isn't started with root privileges.
 #uname=
+uname=murmur
 
 # If this options is enabled, only clients which have a certificate are allowed
 # to connect.
