--- scripts/murmur.ini.orig	2010-04-15 20:49:02.190090655 +0200
+++ scripts/murmur.ini	2010-04-15 20:49:47.077091064 +0200
@@ -1,6 +1,6 @@
 # Path to database. If blank, will search for
 # murmur.sqlite in default locations or create it if not found.
-database=
+database=/var/lib/murmur/murmur.sqlite
 
 # If you wish to use something other than SQLite, you'll need to set the name
 # of the database above, and also uncomment the below.
@@ -48,10 +48,10 @@
 
 # Murmur default to logging to murmur.log. If you leave this blank,
 # murmur will log to the console (linux) or through message boxes (win32).
-#logfile=murmur.log
+logfile=/var/log/murmur
 
 # If set, murmur will write its process ID to this file.
-#pidfile=
+pidfile=/var/run/murmurd.pid
 
 # The below will be used as defaults for new configured servers.
 # If you're just running one server (the default), it's easier to
@@ -123,7 +123,7 @@
 
 # If murmur is started as root, which user should it switch to?
 # This option is ignored if murmur isn't started with root privileges.
-#uname=
+uname=murmur
 
 # If this options is enabled, only clients which have a certificate are allowed
 # to connect.
