--- mumble-1.1.8/src/murmur/murmur_pch.h.debug	2009-03-23 00:05:01.000000000 +0900
+++ mumble-1.1.8/src/murmur/murmur_pch.h	2009-05-17 01:02:54.000000000 +0900
@@ -43,7 +43,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
-#include <sys/types.h>
+#include <linux/types.h>
 #include <sys/socket.h>
 #ifdef Q_OS_LINUX
 #include <sys/capability.h>

--- mumble-1.2.2/compiler.pri.orig	2010-04-01 21:04:58.109466628 +0200
+++ mumble-1.2.2/compiler.pri	2010-04-01 21:05:34.933231938 +0200
@@ -75,8 +75,8 @@
 
 unix {
 	DEFINES *= RESTRICT=__restrict__
-	QMAKE_CFLAGS *= -Wfatal-errors -Wshadow -Wconversion -Wsign-compare -fvisibility=hidden
-	QMAKE_CXXFLAGS *= -Wfatal-errors -Wshadow -Woverloaded-virtual -Wold-style-cast -Wconversion -Wsign-compare -fvisibility=hidden
+	QMAKE_CFLAGS *= -Wshadow -Wconversion -Wsign-compare -fvisibility=hidden
+	QMAKE_CXXFLAGS *= -Wshadow -Woverloaded-virtual -Wold-style-cast -Wconversion -Wsign-compare -fvisibility=hidden
 
 	CONFIG(opt-gcc) {
 		QMAKE_CC = /opt/gcc/bin/gcc
